docs/cri/crictl.md

```diff
@@ -182,33 +182,148 @@ $ crictl info
       }
     ]
   },
+  "cniconfig": {
```

Comment:
containerd is using the default configuration, the output of crictl info looks large